.travis.yml

```diff
@@ -1,25 +1,53 @@
 language: python
-python:
-  - "3.8"
-  - "3.7"
-  - "3.6"  
-  - "3.8-dev"
-matrix:
+os: linux
+dist: bionic
+jobs:
   include:
     # i386: Intel 32-bit
     - name: i386
-      language: minimal
+      language: shell
```

Comment:
https://docs.travis-ci.com/user/languages/minimal-and-generic/#aliases
> Setting the language key to bash, sh or shell is equivalent to language: minimal.

Just note `language: shell` is an alias of `language: minimal` according to the document.

Reply:
Yes. I changed that, together with `matrix:` to `jobs:`, `user:` to `username:`, according to the "Build config validation" information: https://travis-ci.com/taoliu/MACS/builds/150405009/config.